packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -45,6 +52,38 @@ interface EnqueueTaskParams {
   name: string;
 }
 
+/**
+ * A pino-based singleton logger for use within Google Cloud Functions.
+ * Provides a metric() method to log metrics-related data
+ */
+export class GCFLogger {
```

Comment:
Is there a reason for this to exist as a class?  I'd rather have a `getLogger()` method which is directly exported, unless there is a reason for encapsulation, or private data in the class instance.

Reply:
As @bcoe and @chingor13 mentioned above, we want to wrap Pino and abstract it away, hence the GCFLogger class. The purpose of this class is not apparent right now since I haven't actually hidden Pino completely here, but I'll be doing that on the next commit